resources/user-agents/crawlers/social-networking/tumblr/tumblr-agent-generic.json

```diff
@@ -16,6 +16,9 @@
         "Crawler": "true"
       },
       "children": [
+        {
+          "match": "Tumblr/* CFNetwork/* Darwin/*"
```

Comment:
This was added to prevent these from being detected as CFNetwork:

>Tumblr/141 CFNetwork/672.1.14 Darwin/14.0.0
Tumblr/422 CFNetwork/711.5.6 Darwin/14.0.0